packages/server/src/api/controllers/row/ExternalRequest.ts

```diff
@@ -681,6 +681,12 @@ export class ExternalRequest {
       config,
       table
     )
+    //if the sort column is a formula, remove it
+    for (let sortColumn of Object.keys(sort || {})) {
+      if (table.schema[sortColumn].type === "formula") {
```

Comment:
Might just want to add optional chaining in like this (`table.schema[sortColumn]?.type`) as the user could send up a sort column that does not exist in the schema, which would error out without optional chaining.

Reply:
Good spot - done!